docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -339,24 +337,23 @@ Maximum message timestamp difference is the maximum difference allowed between t
 [cols="25%,75%"]
 |===
 h|Editable
-|No
+|Yes
```

Comment:
Is this change taking effect on non-trial Kafka Instances? On trial ones it's not Editable by default...

Reply:
@showuon Please confirm.

Reply:
@mmusil Ack, changing back to No.